src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -79,6 +79,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Degrees = _library.Add(new DegreesFunction());
         public static readonly TexlFunction DegreesT = _library.Add(new DegreesTableFunction());
         public static readonly TexlFunction DropColumns = _library.Add(new DropColumnsFunction());
+        public static readonly TexlFunction EncodeUrl = _library.Add(new EncodeUrlFunction());
```

Comment:
>public static readonly TexlFunction EncodeUrl = _library.Add(new EncodeUrlFunction()); [](http://example.com/codeflow?start=8&length=86)

@carlosfigueira How risky would be to add this function to the core list? Can we get the JS implementation?

Reply:
PA, PFx, and PMs are in sync with this change.